src/AppBundle/Twig/MenuExtension.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace AppBundle\Twig;
+
+use Knp\Menu\ItemInterface;
+use Knp\Menu\Matcher\MatcherInterface;
+use Knp\Menu\Twig\Helper;
+
+class MenuExtension extends \Twig_Extension
+{
+    private $helper;
+    private $matcher;
+
+    public function __construct(Helper $helper, MatcherInterface $matcher)
+    {
+        $this->helper = $helper;
+        $this->matcher = $matcher;
+    }
+
+    public function getFunctions()
+    {
+        return array(
+            new \Twig_SimpleFunction('app_get_current_menu_item', array($this, 'getCurrent')),
```

Comment:
I was unsure if this method is really usefull for other applications (it, at last, uses a not so nice implementation), so I decided to just put it in the AppBundle.